docs/guides/user-containers/readme.md

```diff
@@ -137,3 +137,63 @@ Now any user can select this profile in their Workflow by specifying it in a
 ```bash
 #DW container profile=red-rock-slushy  [...]
 ```
+
+### Using a Private Version of MPI File Utils
+
+If our user's containerized application instead contains `mpirun` and `dcp`,
+because perhaps it's a private copy of [nnf-mfu](https://github.com/NearNodeFlash/nnf-mfu),
+then the administrator would insert two `imagePullSecrets` lists into the
+`mpiSpec` of the NnfContainerProfile for the MPI launcher and the MPI worker.
```

Comment:
It's not really about running a customized version of mpirun/dcp, but rather SCR (or any app really). I envision the private repo to house a containerized version of SCR and that SCR image would be based on nnf-mfu.

The [container example hello world](https://github.com/NearNodeFlash/nnf-container-example/blob/master/src/mpi_hello_world.c) isn't SCR, but gives you an idea of what that custom image might contain.

Reply:
This section is specifically about the mpiSpec.

Reply:
Sure, but I don't understand why were are talking about dcp here.

Reply:
I'll refer to "MPI File Utils" rather than "mpirun" or "dcp"..